### DISCUSSION TITLE
Surface Laptop 3 basically same device as Surface Pro 7 Camera works.

### DISCUSSION THREAD

**Discussion by alex32012** [badstorm/surface-pro-7-opencore#129] [Q&A]
Camera works after USB mapping in windows using USBToolbox shows up as usb device Surface Front Camera

**Comment by Rainringgoaway** [comment]
Maybe sp7 FrontCam isn't driver on Usb side. Unlike Sb3.

**Comment by badstorm** [comment]
Since the conversation is dated I am closing the issue and moving it as a conversation.
